docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -82,7 +87,7 @@ endif::[]
 $ sudo vim /home/ceph-admin/specs/rgw_ingress
 ----
```

Comment:
@fmount Not sure if you saw this question ^.

Reply:
Sorry, I didn't see this question!
It fails because:
1. we removed the ceph_rgw config from haproxy owned by tripleo/director
2. the keystone endpoint is still configured to that haproxy VIP, but the config has been removed
3. we make a request using the cli (e.g. swift list) and we get the following:

```
HTTPConnectionPool(host='10.0.0.103', port=8080): Max retries exceeded with url: /swift/v1/AUTH_852f24425bb54fa896476af48cbe35d3?format=json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc41beb0430>: Failed to establish a new connection: [Errno 111] Connection refused'))
```
It fails to establish the connection, which is expected, and we can proceed with the redeploy.